distributed/scheduler.py

```diff
@@ -3358,8 +3358,14 @@ async def close(self):
             await self.finished()
             return
 
+        async def log_errors(func):
+            try:
+                await func()
+            except Exception:
+                logger.exception("Plugin call failed during scheduler.close")
```

Comment:
Including the plugin name in this message would be nice because as is we just know that some plugin method failed, but not which plugin

Reply:
Ah, thanks for pointing that out @graingert